app/ui/legacy/src/main/res/layout/recipient_alternate_item.xml

```diff
@@ -143,6 +143,15 @@
             tools:visibility="visible"
             />
 
+        <ImageView
+            android:id="@+id/button_copy_email_address"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_margin="13dp"
+            android:background="?attr/selectableItemBackground"
+            android:tooltipText="@string/copy_action"
```

Comment:
Nice :+1:

However, this only seems to be available on API 26+. So we'll still want to use [`TooltipCompat.setTooltipText()`](https://developer.android.com/reference/androidx/appcompat/widget/TooltipCompat#setTooltipText(android.view.View,java.lang.CharSequence)).